src/encoded/static/components/publication.js

```diff
@@ -45,6 +104,21 @@ const PublicationComponent = (props, reactContext) => {
                 </div>
             : null}
 
+            {context.datasets && context.datasets.length ?
+                <SortTablePanel title="Datasets">
+                    <FetchedData ignoreErrors>
```

Comment:
Since this is new code, you can remove `ignoreErrors`. That’s something we used to use to get around an occasional rendering problem, but that problem went away (I don’t remember why it went away).

Reply:
corrected